source/help/00README.md

```diff
@@ -123,21 +123,21 @@ You can bundle non-standard or custom fonts with your submission and instruct `d
 
 `dvips -u+./myfonts.map`
 
-by adding a file called 00README.XXX to your submission with the directive:
+by adding a file called `00README.XXX` to your submission with the directive:
 
 ```
 myfonts.map fontmap
 ```
 
 which identifies your private font map file as a `dvips` fontmap. For map file syntax consult the dvips info pages. Many font bundles from [CTAN](http://www.ctan.org/) come with their custom font map files, and you can use these as is with this directive. The file name of the font map file must have extension "`.map`" and it must consist of letters `A-Z, a-z` only.
 
-## 00README JSON file format
+## `00README` JSON file format
 
-The updated 00README file drops any ambiguity of automatic detection (of the correct compiler, or the correct file to be compiled) and requires a submission to provide sufficient and complete information for the LaTeX compilation process. The submission process will ensure that a complete 00README is included, even if you didn’t provide one initially.
+The updated 00README file drops any ambiguity of automatic detection (of the correct compiler, or the correct file to be compiled) and requires a submission to provide sufficient and complete information for the LaTeX compilation process. The submission process will ensure that a complete `00README` is included, even if you didn’t provide one initially.
```

Comment:
Should the first instance of 00README get the same quotation treatment as the other instances?

No need for comma in first sentence.

Last sentence: change "didn't" to "did not".